extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -231,7 +237,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @return {!Promise|undefined}
    * @private
    */
-  onFetchRemoteConfigSuccess_() {
+  onConfigRewriteSuccess_() {
     this.config_ = this.mergeConfigs_();
```

Comment:
2 small things can be improved:
1. we are unnecessarily passing values via class member fields.
2. naming `onConfigRewriteSuccess_` is a bit mis-leading, since this method also gets called when config-rewriter is not enabled.

Proposal:
- Move `this.config_= this.mergeConfigs_()` out to `rewriteConfig_` function. Don't store intermediate states as member fields, pass into functions as local vars instead.
- rename `rewriteConfig_` to `processConfigs_`, `onConfigRewriteSuccess_` to `registerTriggers_`

Hope that makes sense.

Reply:
Done, thanks!